nucliadb/nucliadb/models/utils.py

```diff
@@ -22,5 +22,9 @@
 import pydantic
 
 
+class FieldIdString(pydantic.ConstrainedStr):
+    regex = re.compile(r"^[^/]+$")
+
+
 class SlugString(pydantic.ConstrainedStr):
-    regex = re.compile(r"[a-z0-9_-]+")
+    regex = re.compile(r"^[a-z0-9_-]+$")
```

Comment:
WDYT about adding some unit tests that make it clearer what would be some valid and invalid examples of each?